utils/vast_parser.js

```diff
@@ -36,7 +36,7 @@ var VastParser = function() {
    * @returns {object[]} An array containing the ad(s) if ads are found, otherwise it returns null.
    */
   this.parser = function(vastXML, adLoaded) {
-    var jqueryAds =  $(vastXML).find("Ad");
+    var jqueryAds =  OO.$(vastXML).find("Ad");
```

Comment:
I'm confused as to why we need this. Is $ not defined anymore in this scope?  This may be a bigger issue.

Reply:
So this is because $ used to be available in the global scope due to a bug we had. The main plugin code does have a reference to jquery that is passed by the AMC, but this `VastParser` function doesn't have it scope.

I think it's just a matter of doing something:
`var VastParser = function($) {`

And then just pass the reference from the AMC when initializing. This way we avoid modifying all occurrences of $

Reply:
Yes, the '$' it's not available in the scope of the library/util, it seems that it was working before because it was defined at the global scope (window) and it was not detected using the debug page because that web app exposes the jQuery symbol globally...I found this trying to build a custom player.

Reply:
@marcosmx could we just pass the reference with a parameter like on the example above? Even though using `OO.$` works, I think it might be better to avoid globals and it might also be a bit cleaner if we just keep using $

Reply:
sure, let me do the proper changes and I'll let you know once it's done

Reply:
@jpfranco @pilievOoyala please check the new approach